knora/dsplib/utils/stack_handling.py

```diff
@@ -0,0 +1,122 @@
+import json
+import re
+import subprocess
+import time
+from pathlib import Path
+from typing import Optional
+
+import requests
+
+from knora.dsplib.models.helpers import BaseError
+
+# relative path to "knora/dsplib/docker", to make it accessible when dsp-tools is called from another working directory
+docker_path = Path(__file__).parent / Path("../docker")
+
+
+def start_stack(
+    max_file_size: Optional[int] = None,
+    enforce_docker_system_prune: bool = False,
+    suppress_docker_system_prune: bool = False
```

Comment:
they are mutually exclusive, couldn't they just be represented in one parameter instead of two?

Reply:
Theoretically yes. There are 3 cases:
 - default: ask user
 - `--prune`: prune
 - `--no-prune`: don't prune

This could theoretically be translated into one parameter:
 - default --> `prune: Optional[bool] = None`
 - `--prune` --> `prune: Optional[bool] = True`
 - `--no-prune` --> `prune: Optional[bool] = False`

This single parameter could then be evaluated with 
 - `if prune==None`
 - `if prune==True`
 - `if prune==False`

But I don't particulary like this construct... :-)